core/domain/learner_progress_services.py

```diff
@@ -614,6 +830,163 @@ def _get_filtered_completed_exp_summaries(
     return filtered_completed_exp_summaries, nonexistent_completed_exp_ids
 
 
+def get_all_completed_story_ids(user_id):
+    """Returns a list with the ids of all the stories completed by the
+    user.
+
+    Args:
+        user_id: str. The id of the learner.
+
+    Returns:
+        list(str). A list of the ids of the stories completed by the
+        learner.
+    """
+
+    completed_activities_model = (
+        user_models.CompletedActivitiesModel.get(
+            user_id, strict=False))
+
+    if completed_activities_model:
+        activities_completed = _get_completed_activities_from_model(
+            completed_activities_model)
+
+        return activities_completed.story_ids
+    else:
```

Comment:
Ditto earlier comment. No need of `else`. Directly return empty array.

Reply:
Done.